addons/website/static/tests/tours/website_form_editor_frontend.js

```diff
@@ -148,7 +148,8 @@ registry.category("web_tour.tours").add("website_form_editor_tour_submit", {
     },
     {
         content:  "Check form is submitted without errors",
-        trigger:  "#wrap:has(h1:contains('Thank You!'))"
+        trigger:  "#wrap:has(h1:contains('Thank You!'))",
```

Comment:
IDK if this double space is intended, should we remove that?

Reply:
It was a mistake and full of linter error but keep it like that, it would need a separate commit to fix those linter error so it's not noise in your pr